src/main/java/frc/utils/applicationsutils/OutputFile.java

```diff
@@ -0,0 +1,58 @@
+package frc.utils.applicationsutils;
+
+import frc.robot.constants.DirectoryPathsConstants;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class OutputFile {
+
+    private String name;
+    private File file;
+
+    private static OutputFile SELF_OUTPUT_FILE = new OutputFile("OutputFile");
```

Comment:
```suggestion
    private static File SELF_OUTPUT_FILE = new File("FileErrors");
```

Reply:
What do you mean by this?

Reply:
this an output for erros with functions on files